libraries/shared/src/BlendshapeConstants.h

```diff
@@ -123,6 +113,24 @@ enum class LegacyBlendshpaes : int {
 // * LipsUpperOpen (not in ARKit)
 // * LipsLowerOpen (not in ARKit)
 
+const QMap<QString, QPair<QString, float>> BLENDSHAPE_SYNONYMS_MAP {
```

Comment:
QMaps are red black trees (not constant lookup).  could use a QHash or std types.

Reply:
(I still think this should be a hash table)

Reply:
The line reference above is no longer in the code. Instead, there's an `extern const QMap <...> READYPLAYERME_BLENDSHAPES_MAP` after a pre-existing `extern const QMap<...> BLENDSHAPE_LOOKUP_MAP`. Both could perhaps be changed to `extern const QHash` though I note that `const QMap` isn't uncommon in other parts of the code.

Reply:
yeah, it’s not uncommon because this is a common mistake 😂.  I bet the majority of the instances of QMap in the code would benefit from being QHashes